### DISCUSSION TITLE
Did you drop support for versions below Xcode 15.3 in version 8.0.0?

### DISCUSSION THREAD

**Discussion by r-plus** [onevcat/Kingfisher#2288] [Q&A] [answered]
on version 8.0.0 of Kingfisher using `nonisolated(unsafe)` that can use Swift 5.10+ (Xcode 15.3).

https://github.com/onevcat/Kingfisher/blob/06b5580411d5a74a460410c25382c221eff57748/Sources/Image/Image.swift#L48

Xcode 15.2 (Swift 5.9) could not compile Kingfisher v8.0.0.

Is this intentional? If so, would it be possible to mention it clearly in the release notes?

**Comment by onevcat** [comment]
It should be unintended and pretty easy to fix. Let me try.

**Comment by onevcat** [comment, marked as answer]
Fixed in 8.0.1.

**Comment by r-plus** [comment]
Thanks!
8.0.1 is compile able :)
